### PR TITLE
Make WLS and FMW Dii samples to support  WIT/WDT download URL

### PR DESCRIPTION
**Pull request by maggiehe00**
Use direct WDT/WIT download URL
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/13479/

In the test out of this run " @@  !!! DEBUG WDT_INSTALL_ZIP_URL is 'https://objectstorage.us-phoenix-1.oraclecloud.com/n/weblogick8s/b/wko-system-test-files/o/weblogic-deploy-main.zip'
@@ Info:  Downloading https://objectstorage.us-phoenix-1.oraclecloud.com/n/weblogick8s/b/wko-system-test-files/o/weblogic-deploy-main.zip

Use latest version:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/13481/

"@@  !!! DEBUG WDT_INSTALL_ZIP_URL is 'https://github.com/oracle/weblogic-deploy-tooling/releases/latest/download/weblogic-deploy.zip'
@@ Info:  Downloading https://github.com/oracle/weblogic-deploy-tooling/releases/latest/download/weblogic-deploy.zip "

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java [hunk 1]
```diff
@@ -45,9 +52,10 @@
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
 import static oracle.weblogic.kubernetes.utils.ApplicationUtils.callWebAppAndWaitTillReady;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkPodReadyAndServiceExists;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getActualLocationIfNeeded;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getNextFreePort;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.testUntil;
-import static oracle.weblogic.kubernetes.utils.CommonTestUtils.withLongRetryPolicy;
+//import static oracle.weblogic.kubernetes.utils.CommonTestUtils.withLongRetryPolicy;
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java:58]: remove commented code

**Review by anpanigr — COMMENTED** [review]
See if we have to change any public documentation for for the usage of new environment  variable.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java (hunk 1 above)
Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java]: Done.

Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java]: Pani, this is the doc: https://oracle.github.io/weblogic-kubernetes-operator/samples/domains/domain-home-in-image/. As you can see "Configuration parameters" section, wdtVersion and witVersion are configurable parameters for the customer, which I didn't change in this PR (per our discussion). So I don't think I need to change change any public documentation.

**Review by anpanigr — APPROVED** [review]

**Pull request merged**